nirc_ehr/resources/queries/study/casesTemp.js

```diff
@@ -0,0 +1,67 @@
+
+var console = require("console");
+var LABKEY = require("labkey");
+
+var prevRow;
+
+function init() {
+    // Get boundary rows for ETL batches
+    if (extraContext.dataSource === "etl") {
+        LABKEY.Query.executeSql({
+            schemaName: 'study',
+            sql: "SELECT Id, date, category FROM study.casesTemp ORDER BY objectid DESC LIMIT 1",
+            maxRows: 1,
+            scope: this,
+            failure: LABKEY.Utils.getCallbackWrapper(function (response) {
+                if (response && response.errors) {
+                    console.log("Error querying boundary case: " + response.errors);
+                }
+            }),
+            success: function (results) {
+                if (results.rows && results.rows.length > 0) {
+
+                    var resultRow = results.rows[0];
+                    prevRow = {};
+                    prevRow.category = resultRow.category;
+                    prevRow.Id = resultRow.Id;
+                    prevRow.date = resultRow.date;
+                }
+            }
+        });
+    }
+}
+
+function getEnddate(row, enddate){
+    var closeDate = new Date(enddate);
+
+    if (row.enddate) {
+        var death = new Date(row.enddate);
```

Comment:
Is this always a death date? Or could it be some other kind of ending date?

Reply:
death or final departure. I updated the variable name